src/orchestration/Orchestration.ts

```diff
@@ -159,6 +164,15 @@ export interface Config {
     telemetries: Telemetry[];
     useBeacon: boolean;
     userIdRetentionDays: number;
+    /**
+     * Apply source maps to error stacks. Enabling this feature will make fetch calls
+     * for javascript source and map files. If these files are cross domain then CORS
+     * headers must be included in their responses. If credentials or other headers are
+     * required then provide a configured fetch function that returns the response as
+     * a string promise.
+     */
+    sourceMapsEnabled?: boolean;
```

Comment:
Issue: to clarify, does this PR actually do two things? 

1. Enhance stack traces of all JSErrors by taking on dependency `stacktrace-js`
2. Optionally enable source maps if `sourceMapsEnabled` and `sourceMapsFetchFunction` are supplied. 

Question: How are the source maps retrieved if `sourceMapsFetchFunction` is undefined?  If supplying `sourceMapsFetchFunction` is required to enable this feature, then `sourceMapsEnabled` may be redundant?

Reply:
Setting `sourceMapsEnabled` to `true` is all that is needed unless additional headers or credentials are needed when requesting the js and map file. `sourceMapsFetchFunction` is optional, users would only provide a fetch method if they need to provide additional headers or credential when the stacktrace lib requests the js and map files.

Reply:
Nit / backlog item (blocking): Before merging, we need to make it clear that this is a _client side_ implementation maps: that is, update configuration names, pr title / description, and update documentation outlining the performance tradeoffs. 

RUM service team also needs to make sure the names of your new feature do not clash with any server-side implementation in the future.

Reply:
No action needed, we can clean it up